src/Common/Models/RobotModel.cs

```diff
@@ -155,4 +155,63 @@ public bool IsValid()
         }
         return true;
     }
+    /// <summary>
+    /// Set the topic to be enabled.
+    /// </summary>
+    /// <param name="topic"></param>
+    public void EnableRosTopic(RosTopicModel topic)
```

Comment:
Move function to `RosTopicModel` name it `Enable()`.

Reply:
renamed and moved to RosTopicModel